tests/test_venue_request_v2.py

```diff
@@ -2237,206 +2226,180 @@ def test_post_decision_stage(self, client, test_client, selenium, request_page,
+        revision_stage_note = test_client.post_note(openreview.Note(
+            content={
+                'submission_revision_name': 'Camera_Ready_Revision',
+                'submission_revision_start_date': start_date.strftime('%Y/%m/%d'),
+                'submission_revision_deadline': due_date.strftime('%Y/%m/%d'),
+                'accepted_submissions_only': 'Enable revision for accepted submissions only',
+                'submission_author_edition': 'Allow reorder of existing authors only',
+                'submission_revision_remove_options': ['keywords']
+            },
+            forum=venue['request_form_note'].forum,
+            invitation='{}/-/Request{}/Submission_Revision_Stage'.format(venue['support_group_id'], venue['request_form_note'].number),
+            readers=['{}/Program_Chairs'.format(venue['venue_id']), venue['support_group_id']],
+            referent=venue['request_form_note'].forum,
+            replyto=venue['request_form_note'].forum,
+            signatures=['~SomeFirstName_User1'],
+            writers=[]
+        ))
+        assert revision_stage_note
 
-#         # Post another revision stage note
-#         now = datetime.datetime.utcnow()
-#         start_date = now - datetime.timedelta(days=2)
-#         due_date = now + datetime.timedelta(days=5)
-#         revision_stage_note = test_client.post_note(openreview.Note(
-#             content={
-#                 'submission_revision_name': 'Camera_Ready_Revision',
-#                 'submission_revision_start_date': start_date.strftime('%Y/%m/%d'),
-#                 'submission_revision_deadline': due_date.strftime('%Y/%m/%d'),
-#                 'accepted_submissions_only': 'Enable revision for all submissions',
-#                 'submission_author_edition': 'Allow reorder of existing authors only',
-#                 'submission_revision_remove_options': ['keywords']
-#             },
-#             forum=venue['request_form_note'].forum,
-#             invitation='{}/-/Request{}/Submission_Revision_Stage'.format(venue['support_group_id'], venue['request_form_note'].number),
-#             readers=['{}/Program_Chairs'.format(venue['venue_id']), venue['support_group_id']],
-#             referent=venue['request_form_note'].forum,
-#             replyto=venue['request_form_note'].forum,
-#             signatures=['~SomeFirstName_User1'],
-#             writers=[]
-#         ))
-#         assert revision_stage_note
+        helpers.await_queue()
+        process_logs = client.get_process_logs(id=revision_stage_note.id)
+        assert len(process_logs) == 1
+        assert process_logs[0]['status'] == 'ok'
 
-#         helpers.await_queue()
-#         process_logs = client.get_process_logs(id=revision_stage_note.id)
-#         assert len(process_logs) == 1
-#         assert process_logs[0]['status'] == 'ok'
+        # revision_invitation = openreview_client.get_invitation(f'''V2.cc/2030/Conference/Submission1/-/Revision''')
```

Comment:
This invitation shoud be expired. Does expiring the super invitation not expire the child invitations anymore?

Reply:
no, we need to expire them one by one :(